src/components/Annotation.tsx

```diff
@@ -6,76 +6,68 @@ import { GenericObject } from "./types/generalTypes"
 
 const interactivityFns = ["onDragEnd", "onDragStart", "onDrag"]
 
-class SemioticAnnotation extends React.Component<AnnotationProps, null> {
-  render() {
-    const { noteData: baseNoteData } = this.props
-    const { screenCoordinates } = baseNoteData
+export default (props: AnnotationProps) => {
```

Comment:
```suggestion
export default function SemioticAnnotation(props: AnnotationProps) {
```

For the sake of correct display names in react devtool and an easier way to navigate across the codebase, it is better to keep the name of function components. As a general rule, I use function declaration on the module level, so the functions are always named and `export default` can be easily applied to them as well

Reply:
Sounds reasonable @alexeyraspopov.

Reply:
Good point, thanks for that.